lib/components/Actions/Button/index.test.tsx

```diff
@@ -28,3 +29,77 @@ test("Button is temporarily disabled when onClick is a promise until the promise
   await new Promise((resolve) => setTimeout(resolve, 100))
   expect(button.attributes.getNamedItem("disabled")).toBeNull()
 })
+
+test("Button renders with different variants", () => {
```

Comment:
We check visual things with visual tests. Why do we need to test it here?

Reply:
Probably not? I'll look into them tomorrow. But hey, just two comments in a 641 lines PR that was generated in 10 minutes? Not bad.

Reply:
Removed this text and the next one, which were too presentational. I've added a note on the `.cursorrules` file so future tests are about behavior instead :)